addons/spreadsheet_dashboard/static/tests/dashboard/dashboard_action_test.js

```diff
@@ -190,3 +191,32 @@ QUnit.test(
         assert.equal(year.placeholder, "Select year...");
     }
 );
+
+QUnit.test("Can delete record tag in the filter by hitting Backspace", async function (assert) {
+    const spreadsheetData = {
+        globalFilters: [
+            {
+                id: "1",
+                type: "relation",
+                label: "Relation Filter",
+                modelName: "product",
+                defaultValue: [37],
+                automaticDefaultValue: true,
+            },
+        ],
+    };
+    const serverData = getServerData(spreadsheetData);
+    const fixture = getFixture();
+    await createSpreadsheetDashboard({ serverData });
+    const tags = fixture.querySelector(".o_control_panel_actions div.o_field_tags");
+    await click(tags);
+    await nextTick();
+    // const selection = fixture.querySelector("div.o_field_many2many_selection");
+
+    assert.equal(fixture.querySelectorAll("o_tag").length, 1);
```

Comment:
```suggestion
    assert.equal(fixture.querySelectorAll(".o_tag").length, 1);
```
its a class, not an html tag ;-)

Reply:
also, I think your selector should be more precise,  the same class could be used at different places. So maybe something like 
```javascript
assert.equal(tags.querySelectorAll(".o_tag").length, 1);
```